---
layout: hello-default

# using Markdown language, you can customize the following variables!

# these variables are required -------------------------------
banner_title: Niclas Thomas
banner_subtitle: DATA SCIENTIST -- AUTHOR -- CONSULTANT
background_image: /images/lighthouse.jpg

# all of the following variables are optional -----------------
page_url: ncthomas # only include this variable if you aren't building the page to your primary domain 

# favicon
favicon: /assets/ico/favicon.ico

# social media
twitter_username: datasciimmunol # # your twitter handle
github_username:  ncthomas # your github username
linkedin: niclas-thomas-1159bb29 # the part after ("https://www.linkedin.com/in/...")
# facebook_username: <-- your facebook username (the part after "https://www.facebook.com/...")
# google_analytics: <-- your google analytics Tracking ID

# By declaring these variables, you can include a button that links to an external website or to media.
# about_button: https://github.com/ncthomas# the link
# about_button_label: get hello # the label on the button

# optional portfolio section ------------------------------------------

# By declaring these variables, you can include a portfolio showcasing your work and organize your portfolio's items into a custom layout, all without adding any CSS. In addition, you must 1) create an HTML file in the_includes folder for each project with the text you'd like to display, and 2) create a YAML file in the _data folder describing the order in which each project should be shown and categorized. See `/includes/example.html` and `/_data/work.yml` for examples.

portfolio:  consultancy # the heading for your portfolio and title of your YAML file
portfolio_description: you can include a portfolio showcasing your work and organize your portfolio's items into a custom layout, all without adding any CSS. # a description to be desplayed below the heading and above the content

# optional CTA section --------------------------------------------------

# By declaring these variables, you can include a CTA section.
cta: contact # the heading for your CTA section
cta_description: Please don't hesitate to get in touch if you'd like to collaborate or get a quote for work. # a description to be desplayed below the heading and above the content
cta_button: mailto:niclas.thomas@gmail.com # a link to an external website or to media
cta_button_label: contact # the label on the button

---			
[//]: # (write a bit about yourself here)

Consultancy work here

### Author of Data Science for Immunologists
  
Data science is a complex subject, but nevertheless one that can be made accessible to all through clear, intuitive explanations and worked examples.


Existing software that forms the backbone of an immunologist's analytical toolkit (such as FlowJo and Prism) are expensive, inflexible and promotes a narrow mindset when it comes to analysing your data. On the other hand, the Python and R programming languages are open source, free and entirely customisable, giving the user the ability to implement any analysis they wish.
​

Although programming languages can seem daunting to the uninitiated, it's far easier to learn than many immunologists may think. Rather than seeking to become an expert programmer, an understanding of the main concepts is more than enough to conduct your own bespoke analyses when coupled with a sound mathematical and statistical understanding.
​

Our new book focusses on the practical aspects of data science, providing sufficient theoretical background without delving into all of the details of each of the methods presented. Introductory chapters are presented alongside the analysis of a publicly available data set, allowing the reader to have practical hands-on experience when learning about important concepts in statistics, machine learning and programming.
​

Once these fundamental topics have been covered, a number of case studies are presented, along with the underlying data, accompanying code and full explanations on topics such as automated, data-driven flow cytometry, building predictive models of disease using gene expression profiling and analysing high throughput sequencing data.